### DISCUSSION TITLE
Implement i18n component

### DISCUSSION THREAD

**Discussion by mochiruntime** [clicepfl/clic-website#241] [Ideas]
- [ ] Write an internationalization component (do we use an existing engine ? Should also consider different rendering techniques such as Markdown sources, maybe delegate to controllers or renderer)
- [ ] Define routing rules to capture languages
- [ ] Write translations for existing pages

**Comment by jonscanzi** [comment]
This is becoming urgent as the more features are implemented in the website the more time-consuming doing this will be. This might be one of the first things to do with new members.

**Comment by mochiruntime** [comment]
Checkout this plugin https://www.i18next.com/

**Comment by mochiruntime** [comment]
Resources for microservices architecture:
- https://nextjs.org/docs/advanced-features/i18n-routing
- https://strapi.io/blog/announcing-content-internationalization-v3-6
